tests/test_ReuseParams_rec.py

```diff
@@ -0,0 +1,98 @@
+from __future__ import print_function
+
+import logging
+logging.getLogger('tensorflow').disabled = True
+import sys
+import os
+sys.path.insert(0, os.path.dirname(os.path.abspath(__file__)) + "/..")
+import unittest
+import numpy.testing
+import better_exchook
+better_exchook.replace_traceback_format_tb()
+
+from Config import Config
+from TFNetwork import *
+from TFNetworkRecLayer import *
+from TFUtil import is_gpu_available
+import TFUtil
+TFUtil.debugRegisterBetterRepr()
+
+
+
+def test_ReuseParams_rec():
```

Comment:
Please do not add this test in a new file. The test should be in the file `tests/test_<filename of ReuseParams>`, i.e. `tests/test_TFNetworkLayer.py`.